packages/bot-web-ui/src/components/quick-strategy/form-wrappers/strategy-description.tsx

```diff
@@ -7,81 +7,85 @@ import { STRATEGIES } from '../config';
 import { TDescriptionItem } from '../types';
 
 type TStrategyDescription = {
-    formfields: React.ReactNode;
-    active_tab: string;
+    formfields?: React.ReactNode;
+    active_tab?: string;
+    tutorial_selected_strategy?: string;
 };
```

Comment:
we could use Partial<Type> of advanced Utility Types TS without '?' mark. I think that would be better, don't you?
```
type TStrategyDescription = Partial<{
    formfields: React.ReactNode;
    active_tab: string;
    tutorial_selected_strategy: string;
}>;
```

Reply:
Sure @maryia-matskevich-deriv